installfest/README.md

```diff
@@ -20,10 +20,17 @@ And listing the pods in the cluster:
 kubectl get pods -A
 ```
 
-Note that the `coredns` pod is stuck in `ContainerCreating`: this is normal as our cluster does not have any CNI installed for now.
+Depending on your local environment and Minikube version, a CNI might or might not have been installed automatically by Minikube.
+Ideally, no CNI has been installed and you will see the `coredns` pod stuck in `ContainerCreating`.
```

Comment:
I believe that if no CNI has been installed, nodes will be marked as `NotReady` and the CoreDNS pods will be stuck in `Pending` state (rather than `ContainerCreating`) since they are unschedulable.